databricks/koalas/namespace.py

```diff
@@ -625,7 +629,9 @@ def read_spark_io(
     return DataFrame(InternalFrame(spark_frame=sdf, index_map=index_map))
 
 
-def read_parquet(path, columns=None, index_col=None, **options) -> DataFrame:
+def read_parquet(
+    path, columns=None, index_col=None, read_pandas_metadata=False, **options
```

Comment:
Maybe we can just name `read_pandas_metadata` to `pandas_metadata` and says it tries to respect the metadata if the Parquet file is written from pandas in the doc.